sherpa/cpp_api/bin/online-recognizer.cc

```diff
@@ -132,6 +132,12 @@ int32_t main(int32_t argc, char *argv[]) {
   }
 
   config.Validate();
+  
+  if ( config.use_gpu ) {
```

Comment:
Could you use
https://github.com/k2-fsa/sherpa/blob/master/scripts/check_style_cpplint.sh
to fix the code style?

---

For instance, it should be
```
  if (config.use_gpu) {
```
instead of
```
  if ( config.use_gpu ) {
```